src/views/uploadEvidence.html

```diff
@@ -0,0 +1,77 @@
+<main id="content" role="main">
+
+  {{> common/betaBanner }}
+
+  {{> common/backLink }}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      {{> common/errorSummary }}
+
+      <h1 id="page-heading" class="heading-large">{{ pageHeading }}</h1>
+
+      {{#if annotations.length}}
+      <h2 class="heading-small">
+        Uploaded files
+      </h2>
+      <dl class="govuk-check-your-answers cya-questions-short cya-questions-fileupload">
+        {{#each annotations}}
+        <div>
+          <dt class="cya-answer">
+            {{this.filename}}
+          </dt>
+          <dd class="cya-change">
+            <a href="{{this.removeAction}}">
+              Remove file <span class="visuallyhidden">{{this.filename}}</span>
+            </a>
+          </dd>
+        </div>
+        {{/each}}
+      </dl>
+
+      {{else}}
+
+      {{#if isWamitabQualification}}
+      {{> upload_evidence/uploadWamitabQualificationDescription }}
+      {{/if}}
+
+      {{/if}}
+
+      <form id="uploadForm" enctype="multipart/form-data" method="POST" action="{{uploadFormAction}}">
+        <input id="defra-csrf-token-for-upload" type="hidden" name="DefraCsrfToken" value="{{DefraCsrfToken}}">
```

Comment:
This should be changed to use the new CSRF token partial

Reply:
done